translation/source/faq.xml

```diff
@@ -12,7 +12,7 @@
 	<string name="howCanIContributeToLishogi">How can I contribute to Lishogi?</string>
 	<string name="lishogiPoweredByDonationsAndVolunteers">Lishogi is powered by donations from patrons and the efforts of a team of volunteers.</string>
 	<string name="findMoreAndSeeHowHelp">You can find out more about %1$s. If you want to help Lishogi by volunteering your time and skills, there are many %2$s.</string>
-	<string name="beingAPatron">Become a patron</string>
+	<string name="beingAPatron">Here are many ways you could contribute to lishogi.</string>
```

Comment:
It would be better to keep the original line and add a new one for the "Here are many ways you could contribute to lishogi", because overwriting a string like this will also overwrite all translations in crowdin